### ISSUE TITLE
Chart visualization err on toggling of traces

### ISSUE THREAD

**Issue by HuifangYeo** [atoti/atoti#227]
## Steps to reproduce

I was toggling the traces of the charts and it suddenly hit error on mouse over.

![visualization_error2](https://user-images.githubusercontent.com/35210495/105809981-1f2b1a00-5fe5-11eb-8ed3-8bf2017a730a.gif)

![image](https://user-images.githubusercontent.com/35210495/105809238-48977600-5fe4-11eb-85fc-8b04b2560241.png)

## Environment

atoti: 0.5.1
Operating System: win32

**Comment by tibdex** [comment]
The `TypeError: Cannot read property 'children' of undefined` is thrown in `useChartTooltipStyle` hook at this line:

```typescript
const targetElement = container.getElementsByClassName("points")[
  hoveredPoint.traceIndex
].children[elementIndex];
```

It can be reproduced by playing [this notebook]( https://github.com/atoti/notebooks/blob/fb567da02791859ed7d26d7f3ece617bc043819e/notebooks/geopricing/main.ipynb).

**Comment by DavidLangva** [comment]
Thanks @tibdex that is indeed the heart of the problem. UI team is now tracking this issue with internal ticket https://support.activeviam.com/jira/browse/UI-5215

**Comment by DavidLangva** [comment]
Update: We merged a fix in earlier today.

**Comment by bloussou** [comment]
Closing this as it is resolved since `0.5.2`
